### PR TITLE
style: address styling feedback

### PR DESCRIPTION
**Pull request by luqven**
## Description
This PR fixes the modal height, upload image preview cropping, filename spacing, and action bar positioning to better match the design comps.

# Before

asset-manager-before
upload-prewview-before
asset-maniger-responsive-before

# After

upload-preview-after
asset-manager-responsive-after
asset-manager-after

https://user-images.githubusercontent.com/16711614/201764958-9ceee78d-57d6-4b7e-85fb-7555e8c1fd31.mov

## Checklist
- [x] Each commit follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec format.
- [x] All existing unit tests are still passing (if applicable).

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #164** <a href="https://app.graphite.dev/github/pr/imgix/contentful/164" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/164?utm_source=stack-comment).

File: src/components/Dialog/Dialog.css
Comment (luqven) [inline src/components/Dialog/Dialog.css:204]: This will need to be reintroduced. Removing it, for now, to make the padding consistent across all the elements.

I will circle back and add break-points-specific padding in a day or two. It will not be merged before it's addressed.

File: src/components/Field/Field.tsx
Comment (luqven) [inline src/components/Field/Field.tsx:34]: This is what resizes the modal window to remove the need to scroll

File: src/components/Gallery/GridImage.tsx
Comment (luqven) [inline src/components/Gallery/GridImage.tsx:24]: This inline style was breaking things.

File: src/components/Gallery/ImageGallery.css
Comment (luqven) [inline src/components/Gallery/ImageGallery.css:19]: Allow the gallery to be scrollable but don't allow the hight to be taller than the space allotted for the grid.

File: src/components/Dialog/Dialog.css
Comment (atlawrie) [reply src/components/Dialog/Dialog.css:204]: Just want to make sure this gets addressed before we merge.

File: src/components/Gallery/ImageGallery.css
Comment (sherwinski) [reply src/components/Gallery/ImageGallery.css:19]: I wonder if notes like these should live in the codebase as comments? That way they are both apparent to the reviewer as well as future maintainers

**Review by atlawrie — CHANGES_REQUESTED** [review]

File: src/components/Gallery/ImageGallery.css
Comment (luqven) [reply src/components/Gallery/ImageGallery.css:19]: I worry that doing so increases work for a maintainer should the commemt become outdated. But happy to add it if we feel it's unlikely to become stale.

**Review by atlawrie — APPROVED** [review]

File: src/components/Dialog/Dialog.css
Comment (luqven) [reply src/components/Dialog/Dialog.css:204]: Discussed offline. Will fix in future PR.

**Pull request merged**
